lib/hanami/action/glue.rb

```diff
@@ -59,7 +59,7 @@ def finish
       #
       # @since 0.4.3
       def sending_file?
-        @_body.is_a?(::Rack::File)
+        @_body.is_a?(::Rack::File) || @_body&.first.is_a?(::Rack::File)
```

Comment:
Rack 2 replaced `#each` method to `Iterator` class ([link](https://github.com/rack/rack/commit/e0ac32913a725899b3ee6bec9f293d6a01469cae#diff-894804c816a11584eb28499cb8bbe396L99)). That's why now action wrap `Rack::File` to array